etc/mercury.conf

```diff
@@ -0,0 +1,4 @@
+
+TODO: Design a syntax method and a C++ library to pull from the info below
```

Comment:
[`yaml-cpp`](https://github.com/jbeder/yaml-cpp/wiki/Tutorial) might be a great option for this. Available in [Conan](https://bintray.com/bincrafters/public-conan/yaml-cpp%3Abincrafters).

Reply:
I recommend this being a `json` file with Conan [parson].